index.js

```diff
@@ -1,39 +1,342 @@
 
-async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+        if (!global.game.user.inventory.find(item => item === "seven days")) {
+          global.game.user.addToInventory("seven days");
+        } else {
+          console.log("You already picked that up");
+        }
+        global.game.user.showInventory();
+        global.game.currentRoom.showInventory();
+      
+        // drop the seven days paper
+      } else if ( input.includes("drop") && (input.includes("paper") || input.includes("seven days")) ) {
+
+        if (global.game.user.inventory.find(item => item === "seven days")) {
+          global.game.user.dropItem("seven days");
+        } else {
+          console.log("You don't have that item!");
+        }
+        global.game.user.showInventory();
+        global.game.currentRoom.showInventory();
+      } else if (input.includes("go") && input.includes("upstairs")) {
+        global.game.moveToRoom('Classroom');
+        inRoom = false;
+      } else if (input.includes("go") && input.includes("outside")){
+        global.game.moveToRoom('Main');
+        inRoom = false;
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how tolet compareSimilarity = (input, triggerWords) => {
+${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
 }
+
+
+async function classroom() {
+  console.log(global.game.currentRoom.message);
+ 
+  let inRoom = true;
+
+  // write Foyer logic block
+  while (inRoom) {
+    console.log(`Player Status: ${global.game.user.checkStatus()}`);
+    let input = await ask('\n>_');
+    input = input.toLowerCase();
+
+    if (commonCommands(input)) {
+      // do nothing, because common command
+    } else {
+
+      // take the seven days paper
+      if ( input.includes("attend lecture") || input.includes("sit") ) {
+        console.log('You sit down in your normal seat. And Josh begins the lecture about recursion.');
+        global.game.user.changeStatus('Confused and Hungry');
+
+        // drop the seven days paper
+      } else if (input.includes("go") && input.includes("downstairs")) {
+        global.game.moveToRoom('Foyer');
+        inRoom = false;
+
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how to ${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
+}
+
+/**** Stand alone objects ****/
+
+let roomNames = {
+  Main: "182 Main Street",
+  Foyer: "182 Main Street - Foyer",
+  Hallway: "Upstairs Hallway",
+  Classroom: "Second floor BCA Classroom"
+};
+
+let roomMessages = {
```

Comment:
You seem to have gotten a little too trigger happy with objects. I'm glad you're not shying away from them, but as I stated on line 47 there's no reason for these to be detached from their objects. It would simplify your code, and improve readability to have all the properties of the rooms on themselves rather than scattered throughout the file.